.eslintrc.json

```diff
@@ -60,7 +79,7 @@
         "import/no-unresolved": 1,
         "no-shadow": "off",
         "linebreak-style": "off",
-        "no-unused-vars": "warn",
+        "no-unused-vars": "off",
```

Comment:
No unused vars should be error or warn, you can off it in several files by eslint directives